src/main/java/bob/commands/Command.java

```diff
@@ -0,0 +1,22 @@
+package bob.commands;
+
+import java.io.IOException;
+
+import bob.data.task.Tasklist;
+import bob.exceptions.BobInvalidDateAndTimeException;
+import bob.exceptions.BobInvalidNumberException;
+import bob.exceptions.BobListIndexOutOfBoundsException;
+import bob.storage.Storage;
+import bob.ui.Ui;
+
+
+/**
+ * Represents command recognisable by Bob.
+ */
+public abstract class Command {
+    public abstract boolean isExited();
```

Comment:
Is it a bit wasteful to create a method that's only for 1 exit command? Can consider using the "instance of" check to exit the app instead.

Reply:
I added this isExit method as I feel that this is more elegant than using instancce of! Its a personal preference that doesn't violate any coding standards hehe.